contribute/release-policy.md

```diff
@@ -0,0 +1,94 @@
+---
+id: release-policy
+title: Release policy
+---
+
+## Release semantics
+
+The Pulsar project follows a variant of [Semantic Versioning](http://semver.org/spec/v2.0.0.html). Existing releases can expect patches for bugs and security vulnerabilities. New features will target minor releases. The difference is that a major version bump will not carry any special meaning in terms of "big features" included in the release or breaking API changes. Instead, it would simply signal a new long-term support (LTS) release.
+
+For example,
+
+* 2.10.0 is a feature release;
+* 2.10.1 is a patch release;
```

Comment:
We need to align on the wording and consistently use it throughout the doc site. 
@codelipenghui and @BewareMyPower mentioned there was no `patch release` term for Pulsar, so we named it `minor` in contrast to `major` in the release process doc. If we decide to go with `feature/patch`, better eliminate `major/minor` to avoid confusing and guessing.

Reply:
Yes, before, we only used major release and minor release. Actually, major release == feature release, minor release == patch release.

+1 to go to feature and patch release.